locales/pt.json

```diff
@@ -89,127 +89,127 @@
+ 	"Error: ": "Erro: ",
+ 	"there are two or more contracts that depend on each other in a cyclic manner": "existem dois ou mais contratos que dependem um do outro de um jeito cíclico",
+ 	"Embark couldn't determine which one to deploy first": "Embark não pôde determinar qual deverá ser publicado primeiro",
+ 	"{{inputName}} has not been defined for {{className}} constructor": "{{inputName}} não foi definido para o contrustor de {{className}}",
+ 	"error deploying %s": "erro ao mostrar %s",
+ 	"executing onDeploy commands": "executando comandos onDeploy",
+ 	"error executing onDeploy for ": "erro ao executar onDeploy para ",
+ 	" does not exist": " não existe",
+ 	"error running onDeploy: ": "erro ao executar onDeploy: ",
+ 	" exists but has been set to not deploy": " existe porém não foi configurado para publicação",
+ 	"couldn't find a valid address for %s has it been deployed?": "não foi possível encontrar um endereço válido para %s , ele foi publicado?",
+ 	"executing: ": "executando: ",
+ 	"the transaction was rejected; this usually happens due to a throw or a require": "a transação foi rejeitada; isso normalmente acontece por causa de um throw ou algum require",
+ 	"no account found at index": "nenhuma conta encontrada no índice",
+ 	" check the config": " verifique a configuração",
+ 	"Both \"from\" and \"fromIndex\" are defined for contract": "Ambos \"from\" e \"fromIndex\" são definidos para contrato",
+ 	"Using \"from\" as deployer account.": "Usando \"from\" como conta de publicação.",
+ 	"{{linkReference}} is too long": "{{linkReference}} é muito longo",
+ 	"{{contractName}} needs {{libraryName}} but an address was not found, did you deploy it or configured an address?": "{{contractName}} precica {{libraryName}} porém o endereço não foi encontrado, você publicou ou configurou um endereço?",
+ 	"attempted to deploy %s without specifying parameters": "tentativa de publicar %s sem parâmetros especificados",
+ 	"deployed at": "publicado em",
+ 	"no contracts found": "nenhum contrato encontrado",
+ 	"Blockchain component is disabled in the config": "O componente Blockchain está desabilitado na configuração",
+ 	"Couldn't connect to an Ethereum node are you sure it's on?": "Não foi possível conectar no nó do Ethereum, você tem certeza que ele está ligado?",
+ 	"make sure you have an Ethereum node or simulator running. e.g '%s'": "verifique que você tem um nó Ethreum ou um simulador executando, por exemplo '%s'",
+ 	"executing": "executando",
+ 	"the transaction was rejected; this usually happens due to a throw or a require, it can also happen due to an invalid operation": "essa transação foi rejeitada; isso ocorre normalmente por causa de um throw ou algum require, também pode ocorrer por alguma operação inválida",
+ 	"Cannot find file %s Please ensure you are running this command inside the Dapp folder": "Não é possível encontrar arquivo %s . Por favor verifique que vocês está executando esse comando dentro do ficheiro DApp",
```

Comment:
`vocês` -> `você`
`ficheiro DApp` -> `directório DApp`

Also I think we can remove the space between `%s .`